tests/test_globus_compute_hello.py

```diff
@@ -0,0 +1,109 @@
+import pathlib
+import subprocess
+
+import pytest
+from globus_compute_sdk import Executor
+from jinja2 import Environment, FileSystemLoader
+
+import chiltepin.configure
+
+
+# Set up fixture to initialize and cleanup Parsl
+@pytest.fixture(scope="module")
+def config(config_file, platform):
+    yaml_config = chiltepin.configure.parse_file(config_file)
+    resources = yaml_config[platform]["resources"]
+    environment = "\n".join(yaml_config[platform]["environment"])
+
+    # with parsl.load(resources):
+    return {"resources": resources, "environment": environment}
+    # parsl.clear()
```

Comment:
Should these comments be removed? I think these were for debugging.
```suggestion
    return {"resources": resources, "environment": environment}
```